### PR TITLE
#12 add properties declared in super types to the list of suggestions

### PR DESCRIPTION
**Pull request by agudian**

### CODE REVIEW

File: org.mapstruct.eclipse/src/org/mapstruct/eclipse/internal/Bindings.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                collectMethodNames( ifc, visited, methodNames );
+            }
+            ITypeBinding superClass = curr.getSuperclass();
+            if ( superClass != null ) {
+                collectMethodNames( superClass, visited, methodNames );
+            }
+        }
+    }
+
+    private static boolean isNotJavaLangObject(ITypeBinding curr) {
```
Comment (gunnarmorling) [inline org.mapstruct.eclipse/src/org/mapstruct/eclipse/internal/Bindings.java:63]: No biggie, but I'd not have the negation "built" into the method name but rather negate it upon usage: `if ( !isJavaLangObject(...)`. Makes it a bit easier to grasp IMO.

**Comment by gunnarmorling** [comment]
Rebased and applied; Thanks, @agudian! Added a commit for the change suggested above. Also added .gitattributes for having Git manage the line ending style automatically.

Btw. do you have any idea how we could test this kind of functionality?

File: org.mapstruct.eclipse/src/org/mapstruct/eclipse/internal/Bindings.java (hunk 1 above)
Comment (agudian) [inline org.mapstruct.eclipse/src/org/mapstruct/eclipse/internal/Bindings.java:63]: Right, that came to be a habit, as in my work projects, there is a (checkstyle|pmd|findbugs|sonar) rule in place that shouts if you always negate the result of one boolean method at all the call sites. I don't mind either way.... :)

**Comment by agudian** [comment]
> Btw. do you have any idea how we could test this kind of functionality?

Yea, a test infrastructure would be great. I'm sure we only need to find out how that is done in the Eclipse projects and do it in a similar way.

**Pull request closed without merging**